packages/zosjobs/src/doc/response/IJob.ts

```diff
@@ -34,6 +34,20 @@ export interface IJob {
      */
     jobname: string;
 
+    /**
+     * flag that holds a job from execution
+     * @type {boolean}
+     * @memberof IJob
+     */
+    hold?: boolean;
+
+    /**
+     * flag that releases a held a job for execution
+     * @type {boolean}
+     * @memberof IJob
+     */
+    release?: boolean;
+
```

Comment:
Is it needed to add the "hold" and "release" properties to the `IJob` interface? I may be missing something here 😢 I'm wondering if we don't need them here since they are defined on `IModifyJobOptions` now.